charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -310,7 +327,16 @@ spec:
       {{- else if .Values.auth.authentication.enabled }}
       authenticationEnabled: "true"
       authenticateOriginalAuthData: "true"
+      {{- if eq .Values.auth.authentication.provider "jwt" }}
+      {{- if .Values.auth.authentication.tls.enabled }}
+      authenticationProviders: "io.streamnative.pulsar.broker.authentication.AuthenticationProviderOIDCToken,org.apache.pulsar.broker.authentication.AuthenticationProviderTls"
```

Comment:
Why `AuthenticationProviderOIDCToken` is configured here?

Reply:
AuthenticationProviderOIDCToken is required, Console-to-Broker and Broker-to-Broker still need token authentication.